patches/newtonsoft-json/0001-Run-dotnet-migrate-to-generate-Newtonsoft.Json.Dotne.patch

```diff
@@ -53,6 +53,7 @@ index 0000000..797f637
 +    <Reference Include="System.Xml.Linq" />
 +    <Reference Include="System" />
 +    <Reference Include="Microsoft.CSharp" />
++    <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.0-alpha-003" PrivateAssets="all"/>
```

Comment:
Does this change introduce building Newtonsoft for full-framework or is this fixing a separate issue?

Reply:
It re-ordered something so that another repo is now using the version that is source-built and was complaining about not being able to build for net45.